low-level/sensors/impl/generic/node_memory_fault.py

```diff
@@ -170,13 +167,9 @@ def run(self):
                 else:
                     self.put_mem_info(self.total_mem)
             else:
-                logger.error("MemFaultSensor: invalid file, shutting down the sensor")
-                self.shutdown()
-                return True
+                raise Exception(f"Invalid file, {mem_path}.")
```

Comment:
`raise with e` as described in the previous file.

Reply:
As justified above.